configure.ac

```diff
@@ -5,6 +5,7 @@ AC_INIT([xrdp], [0.9.6], [xrdp-devel@googlegroups.com])
 AC_CONFIG_HEADERS(config_ac.h:config_ac-h.in)
 AM_INIT_AUTOMAKE([1.7.2 foreign])
 AC_CONFIG_MACRO_DIR([m4])
+AC_GNU_SOURCE
```

Comment:
Why this is needed?

Reply:
Needed to get ``UTMPX_FILE`` and other variables on linux, because of ``__USE_GNU`` in ``utmpx.h``

Reply:
ok